pydatalab/pydatalab/apps/chat/blocks.py

```diff
@@ -81,8 +125,23 @@ def render(self):
                 return
 
         try:
+            model_name = self.data["model"]
+            model_dict = self.data["available_models"][model_name]
+            LOGGER.warning(f"Initializing chatblock with model: {model_name}")
+
+            if model_name.startswith("claude"):
+                self.chat_client = ChatAnthropic(
+                    anthropic_api_key=os.environ["ANTHROPIC_API_KEY"],
+                    model=model_name,
+                )
+            elif model_name.startswith("gpt"):
+                self.chat_client = ChatOpenAI(
+                    api_key=os.environ.get("OPENAI_API_KEY"),
+                    model=model_name,
+                )
+
```

Comment:
probably needs 


```suggestion
                )
            else: 
                raise RuntimeError(f"Unknown model {model_name}")
```

or something similar

Reply:
I'll put in an earlier check that the provided model is in the list of available models